src/main/java/io/cryostat/platform/internal/KubeApiPlatformStrategy.java

```diff
@@ -39,21 +38,18 @@
 
 class KubeApiPlatformStrategy implements PlatformDetectionStrategy<KubeApiPlatformClient> {
 
+    public static final String NO_PORT_NAME = "-";
+    public static final Integer NO_PORT_NUMBER = 0;
```

Comment:
Just curious if we should also use `-` for the empty port number list? I guess we can filter it out first before parsing int.

Reply:
Sure, that's an option too. I don't really mind either way. @ebaron any preference?

Reply:
https://github.com/cryostatio/cryostat-operator/pull/715#issuecomment-1907560164